protoc-gen-swagger/main.go

```diff
@@ -0,0 +1,115 @@
+package main
```

Comment:
That's definitely an interesting idea. From what I've seen in the proto community is that people generally expect different plugins to protoc. Do you know of any examples of people passing flags to change the output to a different language?